m2cgen/assemblers/boosting.py

```diff
@@ -34,6 +36,10 @@ def assemble(self):
                 self.all_trees, self._base_score)
 
     def _assemble_single_output(self, trees, base_score=0):
+        if self._tree_limit is not None:
+            assert self._tree_limit > 0, "Unexpected tree limit"
```

Comment:
I think `__init__` method would be better place for this assert:
- No need to perform it every time this function runs
- It would raise an error right when we pass it, not when we use it later

Reply:
Good point 👍

Reply:
Fixed